share/create_db.pl

```diff
@@ -0,0 +1,119 @@
+use strict;
+use warnings;
+
+use File::Basename qw( basename dirname );
+use File::Spec::Functions qw( catfile );
+use Zonemaster::Backend::Config;
+
+our $MYSQL = "mysql";
+our $PSQL  = "psql";
+
+my $OPT_USERNAME = 0;
+my $OPT_ASKPASS  = 0;
+
+sub usage {
+    print "Usage: " . basename($0) . " [-u] [-p] [-h]\n";
+    print "\n";
+    print "Automatically initialize the Zonemaster database based on the Zonemaster Backend config\n";
+    print "file. Use the values to create the database and a user with enough privileges to\n";
+    print "access this database. Also create all the Zonemaster tables inside this database.\n";
```

Comment:
For FreeBSD this depends on #835 to find the configuration file.

To be consistent: "Zonemaster Backend" --> "Zonemaster-Backend" or "Zonemaster::Backend".